app/domain/audits/filter.rb

```diff
@@ -11,6 +19,16 @@ def audit_status=(value)
                       end
     end
 
+    def allocated_policy
+      if allocated_to == 'no_one'
+        Policies::Unallocated
+      elsif allocated_to.blank?
+        Policies::AllocatedAndUnallocated
```

Comment:
Hi Andrew, not sure about this one (I chose`Policies::AllocatedAndUnallocated` as a way to name a `NilObject`)
But this might be because I am not able to picture your solution in code. What about tracking this idea in its own separate card/PR? If we go for this way of doing things we would also need to update the `AuditedAndNonAudited` policy.